spec/_widget/components/search.spec.js

```diff
@@ -42,10 +42,24 @@ describe('Search', () => {
     expect(trackSearch).toHaveBeenCalledWith('pronounce', ['How to pronounce DNSimple']);
   });
 
+  it('can find results with a truncated query', () => {
+    const results1 = subject.query('hostin');
+    expect(results1.map((r) => r.title).slice(0, 3)).toContain('Web Hosting Support');
+
+    const results2 = subject.query('reg');
+    expect(results2.map((r) => r.title).slice(0, 3)).toContain('Registering a Domain');
+  });
+
+  it('can find results with a truncated query from the dictionary', () => {
+    const results1 = subject.query('expi');
+
+    expect(results1.map((r) => r.title).slice(0, 2)).toContain('Product Expiration Notification');
+  });
+
   describe('queries', () => {
     const queries = {
       'a record': {
-        'Managing A Records': 4,
+        'Managing A Records': 6,
```

Comment:
Most of these results just moved a little bit. I think the trade-off is worth it considering we get completions for partial words.